docs/docs/components.mdx

```diff
@@ -493,7 +493,7 @@ Note: The `feature-dimension` for sequence and sentence features does not have t
   :::
```

Comment:
@d61h6k4 @twerkmeister do we have a place in the docs where we talk about M1 support more broadly? If we don't, I think we should add it (not sure where, maybe you have an idea?) and mention that full M1 support works from 3.6.2 onwards.